styles.css

```diff
@@ -0,0 +1,141 @@
+
+body {
+  background: url(images/background.jpg);
+  background-size: cover;
+
+}
+
+#exphr {
+height: 3px;
+color: #123455;
+background-color: #800000;
+border: none;
+}
+
+#expheader {
+  padding-top: 50px;
+}
+
+#aboutmebg {
+  background: url(none);
```

Comment:
You should consider not using a default background on `body` as well as the use of ids to style.  A better strategy would be to use a class indicating page that is placed on the body tag.  Then you can just add your backgrounds to that without needing to override.  🔨